lib/srv/usermgmt_linux.go

```diff
@@ -211,16 +211,16 @@ func readDefaultKey(key string, defaultValue string) (string, error) {
 	return defaultValue, nil
 }
 
-// readDefaultHome reads /etc/default/useradd for the HOME key,
+// GetDefaultHomeDirectory reads /etc/default/useradd for the HOME key,
 // defaulting to "/home" and join it with the user for the user
 // home directory
-func readDefaultHome(user string) (string, error) {
+func (u *HostUsersProvisioningBackend) GeadDefaultHomeDirectory(user string) (string, error) {
```

Comment:
This not being exposed on the interface made testing on macos annoying, so I moved it over so it could be mocked.